doc/release-notes-1.2.0.rst

```diff
@@ -0,0 +1,550 @@
+
+* :ref:`nrf_bt_scan_readme`:
+
+  * Added an option to update the initial connection parameters.
+
+* :ref:`nus_c_readme`, :ref:`bas_c_readme`, and :ref:`dfu_smp_c_readme`:
+
+  * Fixed an issue where it was not possible to subscribe to the service notifications more than once.
+
+* Updated the :ref:`central_uart` sample to handle data packets that are longer than 212 bytes.
+  Enabled UART flow control to avoid data loss.
+
+* Enabled UART flow control in the :ref:`peripheral_uart` sample to avoid data loss.
+
+* Fixed the :ref:`ble_throughput` sample, which did not achieve the expected throughput measurements.
+
+
+NFC
+---
+
+* Added the following libraries:
+
+  * :ref:`nfc_t4t_cc_file_readme` - reads and parses the Capability Container file that can be found in the Type 4 Tag.
+  * :ref:`nfc_t4t_hl_procedure_readme` - performs the NDEF detection procedure for the Type 4 Tag.
+  * :ref:`tnep_tag_readme` - implements the Tag NDEF Exchange Protocol (TNEP) for a Tag device.
+  * :ref:`tnep_poller_readme` - implements the Tag NDEF Exchange Protocol (TNEP) for a Poller device.
+
+* Added the following samples:
+
+  * :ref:`nfc_tnep_tag` and :ref:`nfc_tnep_poller` -  demonstrate how to use the Tag NDEF Exchange Protocol (TNEP).
+  * :ref:`nrf-nfc-system-off-sample` - demonstrates how to wake the device from System OFF mode using NFC.
+    This sample was ported from the nRF5 SDK.
+
+* Extended the :ref:`nfc_tag_reader` sample with parsing and printing of the Type 4 Tag content.
+
+Multi-Protocol Service Layer (MPSL)
+-----------------------------------
+
+* Added MPSL as a new subsystem.
+  It integrates :ref:`nrfxlib:mpsl` into the |NCS| environment.
+
+
+nRF Desktop
+===========
+
+* Added a ``ble_qos`` module to maintain channel maps.
+
+Build system
+============
+
+* Fixed a bug where a user-defined HEX file that was provided in the static configuration of the :ref:`partition_manager` was not included in the merge operation.
+
+nrfx
+====
+
+* Updated to v2.1.0.
+  For details, see the `changelog <https://github.com/NordicSemiconductor/nrfx/blob/v2.1.0/CHANGELOG.md>`_.
+
+
+??? Where does this go?
+=======================
+
+Settings: Reduced the default partition size for settings storage.
+
+
+Documentation
+=============
+
+* Added or updated documentation for the following samples:
+
+  * nRF9160:
+
+    * :ref:`cloud_client` - added
+    * :ref:`gps_with_supl_support_sample` - added
+    * :ref:`https_client` - added
+    * :ref:`serial_lte_modem` - added
+    * :ref:`nrf_coap_client_sample` - updated
+
+  * Bluetooth Low Energy:
+
+    * :ref:`ble_llpm` - added ??? not mentioned in release notes
+    * :ref:`peripheral_gatt_dm` - added
+    * :ref:`ble_throughput` - updated
+
+  * NFC:
+
+    * :ref:`nrf-nfc-system-off-sample` - added
+    * :ref:`nfc_tnep_poller` - added
+    * :ref:`nfc_tnep_tag` - added
+
+  * Other:
+
+    * :ref:`radio_test` - added
+    * :ref:`timeslot_sample` - added ??? not mentioned in release notes
```

Comment:
Who is responsible for this sample, and in what category should it be listed? Common samples or MPSL subsystem?
@rugeGerritsen

Reply:
@ru-fu . @joerchan and me are responsible for this sample. It demonstrates functionality provided by MPSL. I think "Other" is the correct category for now

Reply:
"This sample demonstrates how to use :ref:`nrfxlib:mpsl` and basic MPSL Timeslot functionality."

Reply:
Agreed to put under MPSL subsystem.